.github/workflows/ci.yml

```diff
@@ -12,36 +12,50 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    container: drupal:9-apache
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
         node-version: [14.x, 16.x, 18.x]
     steps:
       - uses: actions/checkout@v3
+      - name: Install system dependencies
+        run: |
+          apt-get update
+          apt-get install -y git libicu-dev unzip
+          docker-php-ext-install intl
+      - name: Prepare global Drush and PHP configuration
+        run: |
+          mkdir $HOME/.drush
+          cp ./drupal/drush.yml $HOME/.drush
+          echo 'memory_limit = 512M' > /usr/local/etc/php/conf.d/drupal.ini
+      - name: Create Acquia CMS project
+        run: |
+          rm -rf drupal
+          composer create-project --no-install acquia/drupal-recommended-project drupal
+        working-directory: /opt
+      - name: Install Acquia CMS
+        run: |
+          sed -i -e 's!docroot!web!g' composer.json
+          composer install
+          composer require drupal/acquia_cms_common:^1.4 drupal/acquia_cms_starter
+          acms acms:install --no-interaction acquia_cms_headless
+          drush pm:enable acquia_cms_starter --yes
+          chown -R www-data:www-data ./web/sites/default/files
+        working-directory: /opt/drupal
+      - name: Check for success
+        run: |
+          service apache2 start
+          sleep 2
+          curl --fail http://localhost/jsonapi
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Install npm and yarn
+        run: |
+          npm --version
+          npm -g install yarn
       - run: yarn install --frozen-lockfile
       - run: yarn prettier --check .
       - run: yarn run lint
```

Comment:
Something this changes is that it takes much longer to get results from the linters. Could we change this so that the lint is still run before we install ACMS?

Reply:
Good idea. Done.